doc/lightweight-styling.md

```diff
@@ -0,0 +1,47 @@
+# Lightweight Styling
+
+[Lightweight styling](https://learn.microsoft.com/windows/apps/design/style/xaml-styles#lightweight-styling) is a way to customize the appearance of XAML controls by **overriding** the system brushes that they use. Lightweight styling is applied in the Resources of the control itself:
```

Comment:
"default" or "built-in" rather than "system brushes.  Elsewhere, "system brushes" are assumed to be defined as part of the WinUI but that isn't necessarily the case here.

But this doesn't only apply to brushes (system or otherwise)

---

"Lightweight styling is applied in the Resources of the control itself:"

This sounds vague, overly formal, and unnatural.

I'd suggest something like

"Lightweight styles are changed by providing alternate resources with the same key."


The current wording and example suggest that the only way to override a value is directly on each control instance.
But, of course, this isn't the only way to specify a different resource for a lightweight style, as overridden lightweight styles are applied in the same way and with the same priority as any other resource.